doc/guides/west/repo-tool.rst

```diff
@@ -152,62 +148,12 @@ The following three source code topologies supported by west:
 West Structure
 **************
 
-West is currently split in two:
+West's code is distributed via PyPI in a `namespace package`_ named ``west``.
+See :ref:`west-apis` for API documentation.
 
-* Bootstrapper: Installed by ``pip3 install west``, which provides the ``west``
-  binary and the ``west init`` command.
-* Per-installation clone: this is the west repository cloned into each
-  installation, which provides the built-in commands.
-
-.. note::
-
-   This "bootstrapper" / "everything else" separation is similar to the model
-   used by Google's ``repo`` tool, but unfortunately in retrospect was not a
-   good strategy for west.
-
-   In future versions, the ``west`` binary and all built-in commands (including
-   ``init``) will be installed by ``pip3 install west``. Besides eliminating
-   complexity, this will also make it possible to use :ref:`West's APIs
-   <west-apis-west>` from any Python file, not just extension
-   commands.
-
-   Updating west will still be possible manually, e.g. with ``pip3
-   install --upgrade west``. If necessary, it will also still be possible to
-   use different versions of west on the same computer through Python virtual
-   environments.
-
-Bootstrapper
-============
-
-The bootstrapper module is distributed using `PyPI`_ and installed using
-:file:`pip3`. A launcher named ``west`` is placed by :file:`pip3` in the user's
-``PATH``. This the only entry point to west.  It implements a single command:
-``west init``. This command needs to be run first to use the rest of
-functionality included in ``west``, by creating a west installation. The
-command ``west init`` does the following:
-
-* Clones west itself in a :file:`.west/west` folder in the installation.
-* Clones the manifest repository in the folder specified by the manifest file's
-  ``self.path`` section.
-* Creates an initial local configuration file.
-
-Once ``west init`` has been run, the bootstrapper will delegate the handling of
-any west commands other than ``init`` to the cloned west repository. This means
-that there is a single bootstrapper instance installed at any time (unless you
-use virtual environments), which can then be used to initialize as many
-installations as needed, each of which can have a different version of west.
-
-.. _west-struct-installation:
-
-Per-Installation Clone
-======================
-
-A west installation, as described above, contains a clone of the west
-repository in :file:`.west/west`. This is where the built-in command
-implementations are currently provided. The rest of :ref:`West's APIs
-<west-apis-west>` are also currently provided to extension commands by this
-repository. So that west can update itself, the built-in ``west update`` and
-``west selfupdate`` commands fetch and update the :file:`.west/west` repository.
+This distribution also includes a launcher binary, also named ``west``. When
```

Comment:
This is actually not correct on Linux, where the "binary" is actually a Python script

Reply:
I will say "launcher executable"